docs/components/menu/index.html

```diff
@@ -0,0 +1,156 @@
+---
+title: Menu
+assets:
+  - helix-ui.css
+  - helix-ui.js
+---
+{% extends 'component.njk' %}
+{% block content %}
+
+<section>
+  <h2 class="hxSectionTitle" id="menu">Demo</h2>
+  <div class="demo">
+    <hx-disclosure class="hxBtn" aria-controls="menuId">
+      <hx-icon type="cog"></hx-icon>
+      Actions
+      <hx-icon class="menu-icon" type="angle-down"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="menuId">
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  </div>
+  {% code 'html' %}
+    <hx-disclosure class="hxBtn" aria-controls="menuId">
+      <hx-icon type="cog"></hx-icon>
+      Actions
+      <hx-icon class="menu-icon" type="angle-down"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="menuId">
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  {% endcode %}
+</section>
+<!--
+<section>
+<h2 class="hxSectionTitle" id="cog-menu">Cog Menu</h2>
+  <div class="demo">
+    <hx-disclosure class="hxBtn btn-cog" aria-controls="cogmenuId">
+      <hx-icon class="icon" type="cog"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="cogmenuId" >
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  </div>
+  {% code 'html' %}
+    <hx-disclosure class="hxBtn btn-cog" aria-controls="cogmenuId">
+      <hx-icon class="icon" type="cog"></hx-icon>
+    </hx-disclosure>
+    <hx-menu class="hxBox-sm" id="cogmenuId" >
+      <hx-menuitem class="hxBtn hxBtn--link">Action 1</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 2</hx-menuitem>
+      <hx-menuitem class="hxBtn hxBtn--link">Action 3</hx-menuitem>
+    </hx-menu>
+  {% endcode %}
+</section>
+-->
```

Comment:
Are cogs not going to be needed? If it will be needed in the future, I understand the comment block, though I maybe would still just remove the comment block for cleanliness.

Reply:
talked to design and they will be getting back to us on some refactoring, so for now will leave commented until they have updated specs